examples/starwars/droid.bal

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) 2022 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import starwars.datasource as ds;
+
+# An autonomous mechanical character in the Star Wars universe
+distinct isolated service class Droid {
```

Comment:
Do we need to mention `isolated` explicitly? @MaryamZi

Reply:
As this class is not `public` and assuming that it is not exposed publicly via another public construct (e.g., parameter of a public function, field of a public type, etc.), the compiler will infer isolated. So we don't have to explicitly mark it as `isolated`.

If it was public though, you would have to mark it explicitly.